os_dep.c

```diff
@@ -2018,18 +2018,20 @@ void GC_register_data_segments(void)
   GC_INNER ptr_t GC_FreeBSDGetDataStart(size_t max_page_size,
                                         ptr_t etext_addr)
   {
-    word text_end = ((word)(etext_addr) + sizeof(word) - 1)
-                     & ~(word)(sizeof(word) - 1);
-        /* etext rounded to word boundary       */
-    volatile word next_page = (text_end + (word)max_page_size - 1)
-                              & ~((word)max_page_size - 1);
-    volatile ptr_t result = (ptr_t)text_end;
+    volatile ptr_t cap_next_page = NULL;
```

Comment:
This feels like it should be behind a `#ifdef`?

Reply:
fixed in commit [408f667b](https://github.com/capablevms/bdwgc/pull/1/commits/408f667bb59d4d389e44ff70d5631bf922413ed9)